treeshr/TreeCallHook.c

```diff
@@ -89,7 +89,11 @@ static int isEnabled(char *hookName) {
 }
 
 void TreeCallHookFun(char *hookType, char *hookName, ...) {
-  if (!isEnabled(hookName)) return;
+  static int (*TdiExecute)() = NULL;
+  int status;
+  status = LibFindImageSymbol_C("TdiShr", "TdiExecute", &TdiExecute);
+  if (!(isEnabled(hookName) && STATUS_OK)) return;
```

Comment:
Insert:
if ((status & 1) == 0) return;

Reply:
That's what STATUS_OK does. (See Timo's include/status.h)